lib/gilmour/responder.rb

```diff
@@ -157,7 +158,7 @@ def emit_error(extra={})
       # Publish all errors on gilmour.error
       # This may or may not have a listener based on the configuration
       # supplied at setup.
-      if @backend.broadcast_errors
+      if @backend.broadcast_errors != false
         opts[:timestamp] = Time.now.getutc
         payload = {:traceback => @log_stack, :extra => opts}
         publish(payload, Gilmour::ErrorChannel, {}, 500)
```

Comment:
backend.emit_error should call publish depending on the method, not the other way round.